presto-main/src/main/java/com/facebook/presto/execution/NodeTaskMap.java

```diff
@@ -108,87 +147,158 @@ private void addTask(RemoteTask task)
             }
         }
 
-        public PartitionedSplitCountTracker createPartitionedSplitCountTracker(TaskId taskId)
+        public NodeStatsTracker createTaskStatsTrackers(TaskId taskId)
```

Comment:
`createTaskStatsTrackers` drop the trailing `s` maybe since we are returning a singular `NodeStatsTracker` .?